configure

```diff
@@ -150,8 +150,43 @@ if [ $? -eq 0 ] ; then
     fi
 fi
 
+wrf_dir="none"
+
+#
+# If no WRF_DIR environment variable is set, try to locate a WRF build in one
+# of the expected directory names one directory level up; otherwise, try to use
+# the WRF I/O library from the code in $WRF_DIR
+#
+if [ -z "$WRF_DIR" ]; then
+   for d in WRF WRF-4.0.3 WRF-4.0.2 WRF-4.0.1 WRF-4.0 WRFV3; do
+      if [ -e ../${d}/external/io_netcdf/libwrfio_nf.a ]; then
+         echo "Found what looks like a valid WRF I/O library in ../${d}"
+         wrf_dir=$d
+         break
+      fi
+   done
+else
+   if [ ! -e ${WRF_DIR}/external/io_netcdf/libwrfio_nf.a ]; then
+      echo ""
+      echo "Error: The \$WRF_DIR environment variable was set, but the WRF code at"
+      echo "       ${WRF_DIR} doesn't appear to have been successfully compiled"
+      echo ""
+      exit
+   fi
+
+   echo "Using WRF I/O library in WRF build identified by \$WRF_DIR: ${WRF_DIR}"
+   wrf_dir=$WRF_DIR
+fi
+
+if [ $wrf_dir == "none" ]; then
+   echo ""
+   echo " No compiled WRF code found... Check that WRF has been compiled or whatever"
```

Comment:
@mgduda 
Did you mean to leave this as "....or whatever?"

Reply:
That was a test to see how closely anyone checks these PRs. A+. I'll push an amended commit that improves that message.

Reply:
I've just pushed an amended commit. Hopefully the message is more informative now.